src/finch_tracker/tracker_lib.ts

```diff
@@ -23,18 +24,17 @@ export async function fetchChromeSeedData(): Promise<Buffer> {
   return await downloadUrl(kChromeSeedUrl);
 }
 
-// Processes, groups by name and converts to JSON a list of studies.
-export function serializeStudies(
+// Groups studies by name and priority.
+export function groupStudies(
   seedData: Buffer,
   options: ProcessingOptions,
-): Record<string, any[]> {
-  const map: Record<string, any[]> = {};
-  const seed = proto.VariationsSeed.decode(seedData);
-  const addStudy = (path: string, study: proto.IStudy) => {
-    const json = studyToJSON(study);
+): Record<string, Study[]> {
+  const map: Record<string, Study[]> = {};
+  const seed = VariationsSeed.fromBinary(seedData);
+  const addStudy = (path: string, study: Study) => {
     const list = map[path];
-    if (list !== undefined) list.push(json);
-    else map[path] = [json];
+    if (list !== undefined) list.push(study);
+    else map[path] = [study];
```

Comment:
not a fan of these if/else one-liners. Would like these to always have a scoped body `{ ... }` similar to what clang-format now does.

Reply:
agreed, this `else` isn't not the case when a one-liner looks good.
Added {}